src/eval/operation.rs

```diff
@@ -1201,6 +1215,25 @@ impl<R: ImportResolver> VirtualMachine<R> {
             }
             UnaryOp::PushDefault() => Ok(PushPriority::Bottom.push_into(t, env, pos)),
             UnaryOp::PushForce() => Ok(PushPriority::Top.push_into(t, env, pos)),
+            UnaryOp::EmptyRecordWithTail() => match_sharedterm! { t,
+                with {
+                    Term::Record(r) => {
+                        let mut empty = RecordData::empty();
+                        empty.sealed_tail = r.sealed_tail;
+                        Ok(Closure {
+                            body: RichTerm::new(Term::Record(empty), pos), // TODO: is pos right here?
```

Comment:
I think we rather use `pos_op.into_inherited()` or something, meaning that the new record will inherit the position from `%empty_record_with_tail%`. Here it seems it will inherit the position of the argument, but they are surely quite different (one is empty, the other may have fields)?